src/ArborX.hpp

```diff
@@ -14,22 +14,24 @@
 
 #include <ArborX_Config.hpp>
 
-#include <ArborX_Box.hpp>
+#include <ArborX_Geometry.hpp>
+
+// Indexes
 #include <ArborX_BruteForce.hpp>
 #ifdef ARBORX_ENABLE_MPI
 #include <ArborX_DistributedTree.hpp>
 #endif
 #include <ArborX_CrsGraphWrapper.hpp>
 #include <ArborX_LinearBVH.hpp>
-#include <ArborX_Point.hpp>
-#include <ArborX_Sphere.hpp>
+
+// Indexes helpers
+#include <detail/ArborX_AttachIndices.hpp>
+#include <detail/ArborX_HalfTraversal.hpp>
+#include <detail/ArborX_NeighborList.hpp>
```

Comment:
Are there other Experimental things still not included in `<ArborX.hpp>` after these changes?

Reply:
Think so. Node labeling, expand half to full.

Reply:
Then why add these in particular?